.github/workflows/contracts.yaml

```diff
@@ -0,0 +1,104 @@
+name: contracts
+on:
+    push:
+        tags:
+            - "@sunodo/contracts@*"
+    pull_request:
+        paths:
+            - .github/workflows/contracts.yaml
+            - packages/contracts/**
+concurrency:
+    group: ${{ github.workflow }}-${{ github.ref }}
+    cancel-in-progress: true
+permissions:
+    contents: read
+    packages: write
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
+
+            - name: Setup Node.js
+              uses: actions/setup-node@v3
+              with:
+                  node-version: 18
+                  cache: "yarn"
+                  registry-url: https://registry.npmjs.org
+
+            - name: Install Foundry
+              uses: foundry-rs/foundry-toolchain@v1
+
+            - name: Install Dependencies
+              run: yarn
+
+            - name: Build
+              run: yarn build
+              working-directory: packages/contracts
+
+            - name: Generate localhost deployment
+              run: ./dump_localhost.sh
+              working-directory: packages/contracts
+
+            - name: Docker meta
+              id: meta
+              uses: docker/metadata-action@v4
+              with:
+                  images: |
+                      docker.io/sunodo/deployments,enable=${{ github.event_name != 'pull_request' }}
+                      ghcr.io/sunodo/deployments
```

Comment:
Should we name the package as deployments or the container image as contracts?

Reply:
Problem is we've been using deployments all over, in cartesi

Reply:
So let's change the package name for deployments (`./packages/deployments`)

Reply:
But the package is not deployments, it's contracts.
Deployments is a product of contracts.